ts/components/messages/MessageComponent.tsx

```diff
@@ -17,13 +21,14 @@ import {
 import { GlobalState } from "../../store/reducers/types";
 import { convertDateToWordDistance } from "../../utils/convertDateToWordDistance";
 import { formatPaymentAmount } from "../../utils/payment";
+import { paymentRequestTransactionSummaryFromRptId } from "../../store/actions/wallet/payment";
 
 export type OwnProps = MessageDetailsByIdState &
   Readonly<{
     navigation: NavigationScreenProp<NavigationState>;
   }>;
 
-export type Props = OwnProps;
+export type Props = OwnProps & NavigationInjectedProps;
```

Comment:
The component use connect
change NavigationInjectedProps to ReduxProps and
this.props.navigation.dispatch to this.props.dispatch